charts/validator/values.yaml

```diff
@@ -52,6 +52,19 @@ ghost:
   # -- libp2p validate queue size
   libp2pValidateQueueSize: '4096'
 
+  # -- command override for the validator
+  commandOverride: []
+    # - "/usr/local/bin/watchdog"
+    # - "/usr/local/bin/chronicle-app"
+
+  # -- args override for the validator
+  argsOverride: []
+    # - "--log-level=debug"
+    # - "--log-format=json"
+
+  # -- config url for the app, if provided, will override the default config. can be an ipfs url or a http url
+  configsUrl: null
```

Comment:
is null by default. is values supplied here, it will override the defaults and set `-c https://provided-value`

Reply:
null and "" is the same when used as a condition in Go templates

Reply:
but we can remove it

Reply:
removed entirely, since we can leverage commandOverride and argsOverride to achieve this if need be